### PR TITLE
Add UHC Stats documentation in contact summary page.

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

This PR will:
- Add UHC Stats feature documentation in contact summary page.

Ticket: 
- https://github.com/medic/cht-core/issues/6919

Merge after these PRs:
- PR @ cht-core: https://github.com/medic/cht-core/pull/7053 
- PR @ medic-conf: https://github.com/medic/medic-conf/pull/394 

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
Hi @dianabarsan, Can you please take a look at this PR? Thanks!

File: content/en/apps/reference/contact-page.md [hunk 1]
```diff
@@ -21,6 +21,14 @@ Helper variables and functions for the contact summary can be defined in `contac
 | `reports` | An array of reports for the contact. | 
 | `lineage` | An array of the contact's parents (2.13+), eg `lineage[0]` is the parent, `lineage[1]` is the grandparent, etc. Each lineage entry has full information for the contact, so you can use `lineage[1].contact.phone`. | 
 | `targetDoc` | Doc with [`target`]({{< ref "core/overview/db-schema#targets" >}} ) document of the contact, hydrated with the config information of every target it contains a value for. If there is no target document available (for example when viewing a contact that does not upload targets), this value will be `undefined`. This value might also be `undefined` if the contact has not yet synced the current target document. Added in `3.9.0`. |
+| `uhcStats` | Object containing UHC stats information. |
+| `uhcStats.uhcInterval` | Object containing the start and end date of UHC's month interval, it is based on the `uhc.visit_count.month_start_date` defined in the [app settings](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/#app_settingsjson). |
```
Comment (dianabarsan) [inline content/en/apps/reference/contact-page.md:25]: `month interval` -> `reporting period`
`based on the` -> calculated from

File: content/en/apps/reference/contact-page.md [hunk 2]
```diff
@@ -21,6 +21,14 @@ Helper variables and functions for the contact summary can be defined in `contac
...
+| `uhcStats.uhcInterval.start` | Start date timestamp of UHC interval. |
```
Comment (dianabarsan) [inline content/en/apps/reference/contact-page.md:26]: `Start date timestamp of UHC interval` -> `Timestamp, start date of the UHC reporting period.`

File: content/en/apps/reference/contact-page.md [hunk 3]
```diff
@@ -21,6 +21,14 @@ Helper variables and functions for the contact summary can be defined in `contac
...
+| `uhcStats.uhcInterval.end` | End date timestamp of UHC interval. |
```
Comment (dianabarsan) [inline content/en/apps/reference/contact-page.md:27]: Timestamp, end date of the UHC reporting period

File: content/en/apps/reference/contact-page.md [hunk 4]
```diff
@@ -21,6 +21,14 @@ Helper variables and functions for the contact summary can be defined in `contac
...
+| `uhcStats.homeVisits` | Object containing the contact's home visits stats. The [contact's type](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/hierarchy/#app_settingsjson-contact_types) should have `count_visits` enabled and the [UHC visit count settings](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/#app_settingsjson) should be defined, additionally this information is only available for users that have `can_view_uhc_stats` permission and that are not System Administrators. |
+| `uhcStats.homeVisits.lastVisitedDate` | Date timestamp of contact's last home visit. |
```
Comment (dianabarsan) [inline content/en/apps/reference/contact-page.md:29]: `Timestamp, date of contact's last home visit`

I think it's clearer when we start the sentence with the type of value people should expect.

File: content/en/apps/reference/contact-page.md [hunk 5]
```diff
@@ -21,6 +21,14 @@ Helper variables and functions for the contact summary can be defined in `contac
...
+| `uhcStats.homeVisits.count` | Number of contact's home visits. |
```
Comment (dianabarsan) [inline content/en/apps/reference/contact-page.md:30]: `Number of contact's home visits in the current reporting interval`.

**Review by dianabarsan — CHANGES_REQUESTED** [review]
Nice :) 
I left a couple of comments about readability and clarity, and about using commonly used terms (UHCInterval vs reporting period).

File: content/en/apps/reference/contact-page.md (hunk 1 above)
Comment (latin-panda) [reply content/en/apps/reference/contact-page.md:25]: Awesome done!

File: content/en/apps/reference/contact-page.md (hunk 2 above)
Comment (latin-panda) [reply content/en/apps/reference/contact-page.md:26]: Done!

File: content/en/apps/reference/contact-page.md (hunk 4 above)
Comment (latin-panda) [reply content/en/apps/reference/contact-page.md:29]: Agree!

File: content/en/apps/reference/contact-page.md (hunk 5 above)
Comment (latin-panda) [reply content/en/apps/reference/contact-page.md:30]: Done

File: content/en/apps/reference/contact-page.md (hunk 3 above)
Comment (latin-panda) [reply content/en/apps/reference/contact-page.md:27]: Done

**Comment by latin-panda** [comment]
@dianabarsan thanks for the feedback, it's now applied! :) Is it okay?

**Review by dianabarsan — APPROVED** [review]
Nice work!

**Pull request merged**